src/Namotion.Reflection/Context/ContextualAccessorInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Namotion.Reflection
+{
+    /// <summary>Base class for a contextual property or field.</summary>
+    public abstract class ContextualAccessorInfo : ContextualMemberInfo
+    {
+        /// <summary>
+        /// Gets the accessor's type.
+        /// </summary>
+        public abstract ContextualType AccessorType { get; }
+
+        // TODO: Should we implement this?
```

Comment:
I think this convenience property does not make much sense, just use info.AccessorType.Nullability, or info.PropertyType.Nullability, etc.

Reply:
Let's keep it.